src/main/java/de/tum/in/www1/artemis/service/FileUploadExerciseImportService.java

```diff
@@ -40,7 +45,13 @@ public FileUploadExerciseImportService(ExampleSubmissionRepository exampleSubmis
     public FileUploadExercise importFileUploadExercise(final FileUploadExercise templateExercise, FileUploadExercise importedExercise) {
         log.debug("Creating a new Exercise based on exercise {}", templateExercise);
         FileUploadExercise newExercise = copyFileUploadExerciseBasis(importedExercise);
-        return fileUploadExerciseRepository.save(newExercise);
+
+        FileUploadExercise newFileUploadExercise = fileUploadExerciseRepository.save(newExercise);
```

Comment:
Can we move this somewhere into a common method? It's currently duplicated for all exercise types.

Reply:
I moved it into the createExerciseChannel method.